perf/test/angular-cli/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
The performance test doesn't have to be in this PR I think. We don't need that to be on the lower end of our Angular supported versions.

Reply:
I would still want it to follow current Angular CLI project structure. I will upgrade it to Angular latest.